app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::ReviewsController < ApplicationController
+  def index
+    render json: Review.all
+  end
+  def show
+    render json: Review.where({podcast: Podcast.find(params["id"])}), each_serializer: PodcastReviewsSerializer
```

Comment:
I have to admit, this is a bit backwards, but I will allow it. What would be even better is if simply serialized all of the reviews together with the Podcast#show endpoint, and then passed all relevant reviews down from your PodcastShowContainer to your Reviews container